<launch>
  <node pkg="clam_feature_map" type="clam_feature_map" name="clam_feature_map_gen" output="screen">
    <param name="file_path" type="string"
      value="/home/sam/Desktop/kitti07/"/>
    <param name="downsample_size" type="double" value="0.1"/>
    <param name="pcd_name_fill_num" type="int" value="6"/> <!--set 5 for kitti07 and 0 for park-->
  </node>

</launch>